identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources_fr_FR.properties

```diff
@@ -502,6 +502,7 @@ organization.login=Connexion de l'organisation
 organization.name=Nom de l'organisation
 organization.email=E-mail de l'organisation
 invalid.organization.discovery.input=Impossible d'identifier l'organisation. Veuillez fournir le nom de l'organisation pour continuer.
+invalid.organization.discovery.type=Le type de découverte d’organisation donné n’est pas valide ou n’est pas activé.
```

Comment:
I noticed some unusual characters in this text. Could you double-check to ensure everything is correct? If you're using IntelliJ to edit the text, you might want to try using VS Code instead. Also, please check other languages as well.

Reply:
Unusual characters in the github diff should be due to Github using UTF-8 for the UI and some french characters need different encoding. That should be expected.